tutorials/vr/openxr/index.rst

```diff
@@ -0,0 +1,67 @@
+.. _doc_openxr_introduction:
+
+OpenXR Plugin
+=============
+
+Welcome to the Godot OpenXR documentation!
+
```

Comment:
I think for the official documentation we shouldn't be talking about downloading the source and compiling but purely look at obtaining the plugin from the asset library. We don't want to steer normal users to a version in progress.

Compiling or using the latest unofficial build might be the only thing we should leave either in the readme on github or on the wiki on github

Reply:
This section doesn't contain instructions for downloading and compiling the plugin. I left them in the github readme instead since there are not relevant here.

The `Getting Started` section however could be improve to link users to the plugin in the assets directory.
Could you provide a blurb for that section.

Reply:
I'll get something written up this week, slowly getting back into things:)